library/data/recipe/src/main/java/org/quiltmc/qsl/recipe/impl/RecipeImpl.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.recipe.impl;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.item.Item;
+import net.minecraft.recipe.RecipeType;
+import net.minecraft.tag.TagKey;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+import org.quiltmc.loader.api.ModContainer;
+import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;
+import org.quiltmc.qsl.recipe.api.AbstractBrewingRecipe;
+import org.quiltmc.qsl.recipe.api.CustomPotionBrewingRecipe;
+import org.quiltmc.qsl.recipe.api.PotionBrewingRecipe;
+import org.quiltmc.qsl.recipe.api.PotionItemBrewingRecipe;
+
+@ApiStatus.Internal
+public class RecipeImpl implements ModInitializer {
+	public static final Identifier BREWING_ID = new Identifier("quilt_recipe", "brewing");
```

Comment:
For namespaces used in multiple places, please use a constant variable named `NAMESPACE`.